### ISSUE TITLE
💡 [RFC] Topic: Swift and Machine Learning

### ISSUE THREAD

**Issue by donmowry** [SwiftCommunityPodcast/podcast#65]
# Keywords
- Swift
- Python
- Machine Learning
- Dynamic Languages
- Interoperability

# Summary
Swift gained interoperability with scripting languages in Swift 4.2 with the addition of @dynamicMemberLookup. Python, for one, is widely used in machine learning contexts.  

- What are the advantages of using Swift in this field? 
- What other fields could apply these same advantages?
- What other changes would Swift need to improve this interop?
- What are the potential pitfalls of these changes?
- Can Swift replace existing Python solutions in machine learning, for instance? Should it?

In the continuing goal of Swift world-domination, it would be good to discuss the current state of this in a particular field and what the broader implications might mean currently and in the future.
 

# [RFC]
What other information should I include? Who are the experts that could discuss this topic?

**Comment by BasThomas** [comment]
This sounds like a really interesting idea! I think @lattner might know some people that can contribute to this episode :)

**Comment by lattner** [comment]
Yeah, I know a thing or two about this.  I'm happy to let @dynamicwebpaige figure out if something related to S4TF makes sense here.  It might make more sense to wait a bit though.
